mojaloop/account-lookup-service/values.yaml

```diff
@@ -0,0 +1,1248 @@
+    # nameOverride: kafka
+    # host: kafka
+    # port: 9092
+  
+  mysql:
+    ## This is the same as setting the host explicitly, i.e. will set the host as set below. It is included to support importing MySQL charts as a dependency.
+    # fullnameOverride: mysql
+    ## This will override the name of kafka and add the release prefix, e.g. '<RELEASE_NAME>-<nameOverride>'.
+    # nameOverride: mysql
+    # host: mysql
+    # port: 90925
+    # user: central_ledger
+    # database: central_ledger
+    secret: {}
+    #   name: mysql
+    #   key: password
+  
+  adminApiSvc: {}
+    # This is the same as setting the host explicitly, i.e. will set the host as set below. It is included to support importing Kafka charts as a dependency.
+    # fullnameOverride: admin-api-svc
+    ## This will override the name of kafka and add the release prefix, e.g. '<RELEASE_NAME>-<nameOverride>'.
+    # nameOverride: admin-api-svc
+    # host#: admin-api-svc
+    # port: 80
+  
+  config:
+  
+    ## admin and Service Ports 
+    admin_port: 4001
+    api_port: 4002
+  
+    ## Hub Configuration
+    hub_participant:
+      id: 1
+      name: Hub
+  
+    ## Cache configuration
+    cache_enabled: false
+    cache_max_byte_size: 10000000
+    cache_expires_in_ms: 1000
+  
+    ## Health Check endpoint
+    central_services_health_endpoint_param: '/health'
+  
+    ## MongoDB Configuration for Object Store
+    objstore_disabled: true
+    objstore_uri: 'mongodb://$release_name-centralledger-obj:27017/mlos'
+  
+    kafka_partitioner: murmur2_random
+  
+    db_connection_pool_min: 10
+    db_connection_pool_max: 30
+    db_acquire_timeout_millis: 30000
+    db_create_timeout_millis: 30000
+    db_destroy_timeout_millis: 5000
+    db_idle_timeout_millis: 30000
+    db_reap_interval_millis: 1000
+    db_create_retry_interval_millis: 200
+    db_debug: false
+  
+    ## Log Configuration
+    log_level: info
+    log_filter: 'error, warn, info'
+    log_transport: file
+  
+    ## Tracing Configuration
+    event_trace_vendor: mojaloop
+    event_trace_state_enabled: true
+    event_log_filter: 'audit:*, log:warn, log:error'
+    # If set to true, only the metadata object from the event will be printed.
+    event_log_metadata_only: false
+    # A comma-separated list of events that should return immediately instead of waiting for the event promises to resolve
+    # Any combination of: `log,audit,trace`
+    event_async_override: 'log,trace'
+    event_traceid_per_vendor: false
+  
+    # Thirdparty API Config
+    featureEnableExtendedPartyIdType: false
+  
+    ## 
+    display_routes: true
+    run_migrations: false
+    end_point_cache:
+      expiresIn: 180000
+      generateTimeout: 30000
+  
+    ## Error handling Configuration
+    error_handling:
+      include_cause_extension: false
+      truncate_extensions: true
+    resource_versions: 'transfers=1.1,participants=1.1,quotes=1.0'
```

Comment:
Take note that this has changed in v13.1.0 which applies to all ALS services, ref: https://github.com/mojaloop/helm/blob/v13.1.0/.changelog/release-v13.1.0.md#8-upgrade-notes, see section 8.4 for more details.

I don't know if you want to update here, and align to the match the latest updates, or leave that for a separate story?

These changes would also have to be made to ml-api-adapter....so I am leaning to a new story. 

Let me know what you think.

Reply:
yeah I figured the update to 13.1.0 should be separate as this task i.e. v14-v14  is complex enough trying to do the 13.1 updates at the same time .... seems like a bad idea.  So yeah strongly of the opinion that we should have a separate story.